changegen/__main__.py

```diff
@@ -47,16 +48,35 @@ def _get_db_tables(suffix, dbname, dbport, dbuser, dbpass, dbhost):
     show_default=True,
     multiple=True,
 )
+@click.option(
+    "--deletions",
+    help=(
+        "Name of table containing OSM IDs for which <delete> tags "
+        " should be created in the resulting changefile. Table must "
+        " contain <osm_id> column. Can be passed multiple times."
+    ),
+    multiple=True,
+    default=[],
+)
 @click.option(
     "-e",
     "--existing",
     help=(
         "Table of geometries to use when determining whether existing"
         " features must be altered to include linestring intersections."
     ),
-    required=True,
     multiple=True,
 )
+@click.option(
+    "-m",
+    "--modify_meta",
+    help=(
+        "Create <modify> nodes instead of create nodes "
```

Comment:
I can't tell if node in this context refering to an xml node, or an osm node.

Reply:
XML tag. I was fairly consistent about this, I should be saying "tag" instead of node.

Reply:
updated.